package.json

```diff
@@ -185,7 +185,7 @@
         "*.{js,ts,tsx}": "eslint --fix",
         "*.{py,pyi}": [
             "flake8 --select=E9,F63,F7,F82,W605",
-            "black",
+            "black .",
```

Comment:
Context on this change? I thought this command ran for each specific file?

Reply:
good point yeah, the files should be passed to this. for some reason it was never working for me though 🤔